README.md

```diff
@@ -115,6 +115,27 @@ the record will be removed from storage.
 Please note, that setting custom `_ttl` for individual rows is not allowed for tables which
 have a revision retention policy set, as table-wide and row-wide TTLs would conflict.
 
+
+# Options
```

Comment:
Would be nice to expand on this a bit, like what do the different option values stand for.